src/top/utop.el

```diff
@@ -1132,6 +1132,8 @@ See https://github.com/diml/utop for configuration information."))
   ;; Load local file variables
   (add-hook 'hack-local-variables-hook 'utop-hack-local-variables))
 
+(defvar company-backends)
```

Comment:
What happens when the `(defvar company-backends)` in `company.el` is loaded?

Reply:
That will set the value of `company-backends` to the appropriate thing, assuming the user hasn't set it themselves in their config.

The Help page for `defvar` says

> If INITVALUE is missing, SYMBOL's value is not set.

So this `defvar` isn't harmful.